tests/test_full_example_duckdb.py

```diff
@@ -257,22 +257,6 @@ def test_duckdb_arrow_array():
     assert len(df) == 2
 
 
-@mark_with_dialects_including("duckdb")
```

Comment:
test no longer needed with later versions of pandas which seems to deal with typing better